src/style.css

```
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+}
+
+.img {
+  display: block;
+}
+
+.header {
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  display: inline-block;
+}
+
+.nav__item {
+  list-style: none;
+  margin: 0 10px;
+}
+
+.nav__link {
+  display: inline-block;
+  padding: 23px 0;
```

```suggestion
  display: flex;
  height: 60px;
  justify-content: center;
  align-items: center;
```
Ось що я мав на увазі, коли казав, що краще центрувати так, а не падінгами. І клікабельна зона зберігається. На відео продемонстрував


https://user-images.githubusercontent.com/66570142/148383136-883ee862-766f-4c58-b148-2eae34a3cf02.mov